dashboard/app/controllers/api/v1/teacher_feedbacks_controller.rb

```diff
@@ -49,7 +49,8 @@ def count
       User.find(feedback.teacher_id).authorized_teacher?
     end
 
-    render json: @all_unseen_feedbacks.count, each_serializer: Api::V1::TeacherFeedbackSerializer
+    # TODO: is it ok to use the text/plain content type here? We're not actually returning JSON.
```

Comment:
It seems like this API just returns a string literal like "0" or "5", which isn't JSON as far as I know. Wasn't sure if it was relevant to keep the response content type as json or not.

Reply:
I was on maternity leave when this part of the feature was built. Hopefully Erin knows.

Reply:
> It seems like this API just returns a string literal like "0" or "5", which isn't JSON as far as I know.

A string literal like `"5"` is valid JSON text, it parses as the `number` 5. The JSON format itself doesn't require its root `value` to be an `object`, any other `value` type (`number`, `string`, `array`, `true`/`false` or `null`) is valid JSON on its own as well.

(note: a string like `"foo"` is not valid JSON text, it should be `'"foo"'` with quotation marks as part of the string contents.)

Examples:
```bash
$ echo '5' | jq
5
$ echo '[true, false, null]' | jq
[
  true,
  false,
  null
]
$ echo 'foo' | jq
parse error: Invalid literal at line 2, column 0
$ echo '"foo"' | jq
"foo"
```

Reply:
Thanks for clearing that up - I'll change it back.

Reply:
If we're just using this as a display value, I'd guess that we don't really care whether it's a string or a number

Reply:
The count, as far as I know, is used for the notification on the student home page where we say something like "You have X levels with unread feedback", so in that sense it's only a display value, BUT [we do a comparison with the value to determine if it's > than 0 to decide](https://github.com/code-dot-org/code-dot-org/blob/607aac8d0e430c23e3b1e5cb2e19fcd2aaa6cb57/apps/src/templates/feedback/StudentFeedbackNotification.jsx#L40) whether we should show that notification or not so I think we need it to be a number.

Reply:
Thanks @Erin007 . Are you aware of any UI test coverage on that feature where we could add a check that this is working?

Reply:
Tested that this notification still works locally, both when there's no unread feedback, and when there is one:

![image](https://user-images.githubusercontent.com/6564873/92182479-a9899c00-ee00-11ea-8b0c-651b67c102e8.png)

Reply:
Thanks, @uponthesun ! That's good enough for me regarding StudentFeedbackNotification.